.ci/scripts/tag_release.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
```

Comment:
In order to use this particular script is required to configure the workspace with the git context as the current git context is a git detached commit, for such see:
- https://github.com/elastic/apm-agent-rum-js/blob/master/.ci/scripts/prepare-git-context.sh

Reply:
Ah, OK. Interesting. I'll borrow from that script and add it to the top of this one. Good catch here. I wasn't aware of that.

Reply:
@v1v Could you please take a look at my latest commit and see if that lines up with how your script expects to operate?

Reply:
Switched to using the [new git context manager](https://github.com/elastic/apm-pipeline-library/pull/481).